recipes/source.rb

```diff
@@ -31,6 +31,13 @@
 
 Array(node['redis']['source']['pkgs']).each { |pkg| package pkg }
 
+current_version = shell_out("redis-server --version|grep -o -E v=[0-9]+.[0-9]+.[0-9]+").stdout.slice(2..-1)
```

Comment:
```suggestion
current_version = shell_out("redis-server --version").stdout.gsub(/^.*v=(\d+\.\d+\.\d+).*$/, '\1')
```
Ruby使えるのでパイプせずにこれでも多分良いと